crates/ruma-events/src/kinds.rs

```diff
@@ -446,6 +446,20 @@ pub struct StrippedStateEvent<C: PossiblyRedactedStateEventContent> {
     /// This is often an empty string, but some events send a `UserId` to show which user the event
     /// affects.
     pub state_key: C::StateKey,
+
+    /// The globally unique event identifier for the user who sent the event.
```

Comment:
> for the user who sent the event

This doesn't seem right, shouldn't it just be
```suggestion
    /// The globally unique identifier for the event.
```

Reply:
Oops, indeed.

Reply:
That said, I copy-pasted it from another definition :stuck_out_tongue:.

I'm going to fix them in another commit.

Reply:
Actually, that makes me want to fix other docs, I'll do it in a separate PR.